docs/apm/traces/view-and-investigate-traces.md

```diff
@@ -59,23 +59,49 @@ Queries above 7 days may be slower to load.
 
 The results are not automatically updated. If you want to refresh traces, click the refresh button on the top right corner of the page.<br/>![Refresh.png](/img/traces/Refresh.png)
 
-### Trace Duration Breakdown Chart
+### Trace Query Visualizations
 
-The Trace Duration Breakdown Chart helps you understand average trace durations in every time bucket as well as the amount of time each service contributed to the end-to-end duration. Use this chart to:
+The Trace Query Visualizations allow you to visualize the breakdown of where time was spent in traces for various services in your application. By aggregating traces, you can gain insights into anomalies and unexpected behaviors, leading to faster Time-to-Resolution (TTR) and a higher Return on Investment (ROI). It offers six charts to help you analyze and trace data efficiently. These include:
+* Trace duration
+* Error count
+* Error count
+* Span count
+* Span count
+* Trace duration as histogram
 
-* Quickly understand intermittent duration spikes or slowdowns.
-* Immediately spot the offending service by comparing CPC contribution by service.
+<img src={useBaseUrl('/img/traces/breakdown2.png')} alt="breakdown2.png" width="950px" height="500" />
 
-![breakdown](/img/traces/breakdown2.png)
+For each chart type, you can choose to view it as either a **timeseries** or **histogram** chart. Here is some more detailed information about each chart type:
+* **Trace duration**. It allows you to quickly identify services that are causing spikes or slowdowns in trace duration. It helps you understand the average trace duration for every time bucket, as well as the amount of time each service contributed to the end-to-end duration.
```

Comment:
```suggestion
* **Trace duration**. 
Timeseries: It allows you to quickly identify services that are causing spikes or slowdowns in trace duration. It helps you understand the average trace duration for every time bucket, as well as the amount of time each service contributed to the end-to-end duration.
Histogram: helps you understand the distribution of trace durations. For example you can notice groups of durations responsible for longer traces during slowdown periods.
```

Reply:
@pbrzoska-sumo I see you have removed the Trace Duration description and added it in Timeseries.

Reply:
@pbrzoska-sumo : I see you changed the Trace length description to Timeseries.Do we want to leave the Trace duration field blank? Let me know if the below description can be added.
Trace duration: The amount of time each service contributed to the end-to-end duration.